units/src/amount.rs

```diff
@@ -1181,10 +1181,14 @@ impl core::iter::Sum for Amount {
 /// * Show or hide denomination
 /// * Dynamically-selected denomination - show in sats if less than 1 BTC.
 ///
-/// However this can still be combined with [`fmt::Formatter`] options to precisely control zeros,
+/// However, this can still be combined with [`fmt::Formatter`] options to precisely control zeros,
 /// padding, alignment... The formatting works like floats from `core` but note that precision will
 /// **never** be lossy - that means no rounding.
 ///
+/// Note: This implementation is currently **unstable**. The only thing that we can promise is that
+/// unless the precision is changed, this will display an accurate, human-readable number, and the
+/// serialization will round-trip with [`FromStr`]
```

Comment:
We should say "the default serialization (one with unmodified `Formatter` options) will round trip" since this text makes it look like it might roundtrip other things.

Reply:
makes sense, updated